src/Dfe.PlanTech.Web/Views/Shared/Components/Accordion/Default.cshtml

```diff
@@ -0,0 +1,41 @@
+@model IEnumerable<Dfe.PlanTech.Domain.Content.Interfaces.IAccordion>
+
+<div class="govuk-grid-column-one-third">
+
+    <div data-module="appstepnav" id="step-by-step-navigation" class="app-step-nav govuk-!-display-none-print app-step-nav--active" data-show-text="Show" data-hide-text="Hide" data-show-all-text="Show all actions" data-hide-all-text="Hide all steps">
+        <ol class="app-step-nav__steps">
+            @foreach (var accordion in Model)
```

Comment:
I'd personally make the individual accordion section another partial view, but YMMV.